### ISSUE TITLE
Treat unknown properties as warnings (not errors)

### ISSUE THREAD

**Issue by Jellby** [w3c/css-validator#62]
Could we please have an option to treat unknown properties (not detected as vendor extensions) as warnings instead of errors? After all they are perfectly valid, they'll just be ignored if not supported, but warnings are useful in case there's a misspelling.

The following CSS:

```
h3 {
visibility: none;
adobe-hyphenate: none;
hyphens: none;
-epub-hyphenate: none;
}
```

should only give an error for ``visibility: none`` (unvalid value for valid property), the rest are at most warnings, regardless of the CSS level.

**Comment by ylafon** [comment]
Well, in that case, the invalid value will also be considered as non-existent, so a warning as well.
No errors, just warnings.
Note also that warnings are usually ignored, someone making a typo in a property name will probably never notice it if it is a warning and not an error.
